### DISCUSSION TITLE
Can anyone give me the default string of "Format of the YouTube search term"?

### DISCUSSION THREAD

**Discussion by i18wm** [KRTirtho/spotube#385] [Q&A]
when i delete that string, the app doesn't work

**Comment by KRTirtho** [comment]
There's only 3 variables `$MAIN_ARTIST`, `$TITLE` & `$FEATURED_ARTISTS`
you can mix and match those and can use them multiple times
The default string is `$MAIN_ARTIST - $TITLE $FEATURED_ARTISTS`
